dominykasmurauskas/spryker-b2c
# examples/state-machine

To test it:

Go to your shop http://www.de.b2c-demo-shop.local/ and choose a product, add it to the cart, and checkout using the Invoice payment method.
Open the orders page in the Backend Office http://zed.de.b2c-demo-shop.local/sales/ and then open your order. The Demo01 process is now applied on your order.
You will notice that there is a button next to your order called ship, this one triggers the manual event ship.
You can click on the last state name under the state column to see what the current state for a specific item is. The current state has a yellowish background color.
Now click on ship to move the item into the next state.
Click again on the last state name and check the current state.
You can keep moving the item until you reach a state that is not manually executable.
